webpack.config.js

```diff
@@ -243,10 +250,21 @@ if (TARGET !== "readme") {
         )
     };
 
-    var pluginsProd = [
-        // Clean UI/js before building (must be first)
-        new CleanWebpackPlugin(CleanWebpackPluginOptions),
+    // Compile bootstrap module as a virtual one
+    const VirtualModulesPluginOptions = {
+        "./bootstrap.js":
+            `/* eslint-disable */\n` +
+            `define(["dojo/parser","dojo/ready","` +
+            includedRequires.join('","') +
+            `"], function(parser, ready) {\n` +
+            `    ready(function() {\n` +
+            `        console.log('Bootstrapped');\n` +
```

Comment:
this maybe only for dev builds? (the `console.log` I mean)

Reply:
Debugging leftover. Removed.